odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -2536,6 +2536,32 @@ def test_98_prefetch_translate(self):
         self.assertFalse(Model.rare_description.prefetch)
         self.assertFalse(Model.rare_html_description.prefetch)
 
+    def test_99_prefetch_group(self):
+        Model = self.env['test_new_api.prefetch']
+        records = Model.create([{} for _ in range(10)])
+
+        Model.invalidate_cache(ids=records.ids)
+
+        with self.assertQueryCount(1):
```

Comment:
Using `self.assertQuery()` would be explicit about which fields are fetched, which is kind of the point, isn't it?

Reply:
Yes, you are right!